### DISCUSSION TITLE
Power on / off Plan

### DISCUSSION THREAD

**Discussion by VisionLabGroup** [Ysurac/openmptcprouter#3704] [Q&A]
Is it possible to plan the power-on and off of the PC with openmtcp router?

**Comment by AKHwyJunkie** [comment]
I don't think this is a native feature.  However, there is full SSH access, which provides access to reboot and shutdown commands.  So, a shutdown sequence could be initiated via a script.  Startup gets a bit more tricky.  I would look at WOL (wake on LAN) as my preferred approach.

When using OpenMPTCPRouter in a virtual environment, startup and shutdown can be much more easily integrated into that planning, which is a lot more flexible IMO.
